api/cfnetworking/cfnetv1/policy_test.go

```diff
@@ -12,10 +12,7 @@ import (
 
 var _ = Describe("Policy", func() {
 	var client *Client
-
-	BeforeEach(func() {
-		client = NewTestClient()
-	})
+	BeforeEach(func() { client = NewTestClient() })
```

Comment:
usually, go fmt keeps BeforeEach on separate lines. please make sure you ran go fmt. (`bin/test` should run go fmt for you)